Src/PerformanceCollector/Unit.Tests/WebAppPerformanceCollector/RateCounterTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Unit.Tests
+{
+    using Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.WebAppPerformanceCollector;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    [TestClass]
+    public class RateCounterTests
+    {
+        [TestMethod]
+        public void RateCounterGaugeGetValueAndResetWorking()
```

Comment:
nitpick: a little more clarity about what is being tested and what is expected in the test name would be good, I think this might mean that you are testing if the rate counter gauge can get a value and that it is reset when you call to get a value but I'm just guessing.